metaflow/client/core.py

```diff
@@ -938,6 +938,30 @@ def metadata(self):
             for obj in all_metadata
         ]
 
+    @property
+    def origin_pathspec(self):
```

Comment:
For consistency, do you want to add this method to the parent objects as well? That will simplify the logic here as well to `os.path.join(self.parent.origin_pathspec, origin_task_id)`.

Reply:
Agreed it should be on all levels. Also make it clear in the doc that if this isn't a clone, it returns None.